circus-train-comparator/src/test/resources/log4j.xml

```diff
@@ -24,7 +24,7 @@
   </appender>
 
   <logger name="com.hotels.bdp.circustrain">
-    <level value="DEBUG" />
```

Comment:
Instead of the log to warn changes, can we use something like: https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#redirectTestOutputToFile
For Travis builds?

As a dev I kind of like chatty logs when running tests in the IDE, it really helps catching bugs. I don't like having to change it back and forth all the time and accidentally pushing it if I don't pay attention.

Code looks good btw, the log thing is the only issue I got :)

Reply:
I'll try this but the thing is that Travis will kill the job if it doesn't receive output in 10 min. There is a wait to extend the timeout but we may not want to do that